certification/pyxis/layers.go

```diff
@@ -0,0 +1,71 @@
+package pyxis
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+
+	cranev1 "github.com/google/go-containerregistry/pkg/v1"
+
+	log "github.com/sirupsen/logrus"
+)
+
+func (p *pyxisEngine) CheckRedHatLayers(layerHashes []cranev1.Hash) (*CertImage, error) {
+	ctx := context.Background()
```

Comment:
This should be passed in, not created here.

Reply:
creating in validate for now ..